docs/dev/action_collection.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Action Collection
+---
+
+# Action Collection Design
+
+The action collection tracks all actions taken by all Chef resources.  The resources can be in recipe code, as sub-resources of custom resources or
+they may be built "by hand".  Since the action collection hooks the events which are fired from the `run_action` method on Chef::Resource it does
```

Comment:
```suggestion
they may be built "by hand".  Since Action Collection hooks the events which are fired from the `run_action` method on Chef::Resource it does
```